bin/diamond-setup

```diff
@@ -77,7 +78,7 @@ def getCollectors(path):
 
 
 def typeToString(key):
-    if isinstance(obj.config[key], basestring):
```

Comment:
Any of these `basestring` -> `str` translations have lost meaning.

This is covered in `six`, see: https://bitbucket.org/gutworth/six/src/8a545f4e906f6f479a6eb8837f31d03731597687/six.py?at=default#six.py-49